jepsen/src/jepsen/net.clj

```diff
@@ -0,0 +1,24 @@
+(ns jepsen.net
+  "Controls network manipulation."
+  (:use jepsen.control))
+
+(defprotocol Net
+  (drop-from! [net source] "Drop traffic originating from this source IP.")
```

Comment:
This implicitly assumes jepsen.control is telling us what nodes to act on. I'd prefer to make the source and dest nodes explicit; e.g. (drop! [net source dest])